usaspending_api/search/models/base_award_search.py

```diff
@@ -19,6 +19,9 @@ class BaseAwardSearchModel(models.Model):
     category = models.TextField()
     type = models.TextField()
     type_description = models.TextField()
```

Comment:
Discrepancies between materialized view definition and the base class. (looking at `mv_directpayment_award_search`) 

`pop_country_population` vs `pop_county_population` (which one? or both?)
missing `total_covid_obligation`

![image](https://user-images.githubusercontent.com/22781949/98625702-53881780-22cd-11eb-8a1b-0cda1d1ecb18.png)

Reply:
Verified the materialized views schema match each other

Reply:
Removed `pop_country_population` from model and `pop_county_population` from MVs